src/pages/workspaces/workspace/Data.js

```diff
@@ -1379,6 +1379,23 @@ export const WorkspaceData = _.flow(
                   undefined,
                   () =>
                     Utils.cond(
+                      [
+                        isAzureWorkspace && wdsError,
+                        () =>
+                          div(
+                            {
+                              style: { textAlign: 'center', lineHeight: '1.4rem', marginTop: '1rem', marginLeft: '5rem', marginRight: '5rem' },
+                            },
+                            [
+                              'Oh no! Something went wrong while preparing your data tables.',
+                              div({}, [
+                                'Please ',
+                                h(Link, { style: { marginTop: '0.5rem' }, onClick: () => setTroubleshootingWds(true) }, ['check the status']),
+                                ' of your data table service and share the details with support@terra.bio to troubleshoot the problem.',
```

Comment:
It would be nice if `support@terra.bio` was a `mailto:` link.

Reply:
I added `mailto` to this and the `WdsTroubleshooter`.  Some slight duplication that might lend itself nicely to a `MailtoLink` component, but I don't trust my TS generalization skills enough to make that effort worth it at the moment.